theme-dev/README.md

```diff
@@ -1,32 +1,33 @@
-# Themer Documentation for the Themer Plugin
+# Themer Documentation
 
-This is the Documentation for the Themer Plugin
+Welcome to the Documentation for the [Themer Plugin](https://github.com/Vendicated/AliucordPlugins/tree/main/Themer)
 
 * [Main Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/MainStrings.md)
 * [Simple Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/SimpleStrings.md)
 * [Accent Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/AccentColors.md)
 * [Primary Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/PrimaryColors.md)
 * [uikit Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/uikitStrings.md)
 * [Drawable Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/DrawableStrings.md)
-* [Random Things which can change over time and also some resources which might help](https://github.com/Aliucord/documentation/blob/main/theme-dev/RandomThings.md)
+* [Material You](/theme-dev/MaterialYou.md)
+* [Random Things which may change over time, and also some resources which might help](https://github.com/Aliucord/documentation/blob/main/theme-dev/RandomThings.md)
 
-<h1 align="Center">Small introduction</h1>
+<h1 align="Center">Minor Introduction</h1>
 
-**DO NOT CHANGE THE JSON FILE DIRECTLY BECAUSE MOST OF THE TIME YOU FUCK SOMETHING UP**
+<h4 align="center">DIRECTLY MODIFY THE JSON FILE AT YOUR OWN RISK, YOU WILL LIKELY FUCK SOMETHING UP</h4>
```

Comment:
use markdown headers, not html headers

Reply:
Is is possible to center this text using markdown though?